statistics/feedback.go

```diff
@@ -585,10 +556,11 @@ func buildNewHistogram(h *Histogram, buckets []bucket) *Histogram {
 
 // queryFeedback is used to serialize the QueryFeedback.
 type queryFeedback struct {
-	IntRanges   []int64
-	HashValues  []uint64 // HashValues is the murmur hash values for each index point.
-	IndexRanges [][]byte
-	Counts      []int64 // Counts is the number of scan keys in each range.
+	IntRanges    []int64
```

Comment:
Can we merge `IntRanges`, `ColumnRanges` and `IndexRanges` into one `[][]byte`?

Reply:
Then it is not compatible with previous versions.